crates/tabby-index/src/code/mod.rs

```diff
@@ -85,7 +85,7 @@ impl IndexAttributeBuilder<SourceCode> for CodeBuilder {
             for await (start_line, body) in CodeIntelligence::chunks(&text, &source_code.language) {
                 let attributes = json!({
                     code::fields::CHUNK_FILEPATH: source_code.filepath,
-                    code::fields::CHUNK_GIT_URL: source_code.git_url,
+                    code::fields::CHUNK_SOURCE_ID: source_code.source_id,
```

Comment:
no need to add this, source_id is already indexed:

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/48d9c084cc80d20bd439102cc0768920f6680e91/crates/tabby-common/src/index/mod.rs?plain=1#L25

Reply:
But it is attribute, not index.
Do you mean that we can just remote this field `CHUNK_GIT_URL`?